common/templates/mathjax_include.html

```diff
@@ -27,6 +27,27 @@
       ]
     }
   });
+   MathJax.Hub.signal.Interest(function(message) {
+    set_mathjax_display_div_settings(message);
+  });
+  function set_mathjax_display_div_settings(message) {
+    if (message[1].id && message[0] == 'End Math') {
+        var status = message[2] == 'Reprocess' ? true : false;
+        if (status) {
+            try {
+                var mathjax_display = message[1].MathJax['preview'].nextSibling;
+                if (mathjax_display.className == 'MathJax_Display') {
+                    var input_id = mathjax_display.parentElement.id.replace("_preview", "");
+                    mathjax_display.setAttribute('aria-label', 'Math output for ' + input_id);
```

Comment:
@Shrhawk the value of input_id is not human readable i.e. what a screen reader user hears is:
"Math output for input_i4x-test-test101-problem-28785b61b5ff42b788987860d25fee94_2_1"
I was hoping you could pull the value of the question text here i.e.
"Math output for Enter the numerical value of Pi"
But judging from your [JIRA comment](https://openedx.atlassian.net/browse/TNL-1072?focusedCommentId=94262&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-94262) it seems like this may not be possible without restructuring the problem type.  The value we are looking for would be the value of ${label} in common/lib/capa/capa/templates/formulaequationinput.html